app/src/main/res/values/strings.xml

```diff
@@ -736,8 +736,11 @@
     <string name="location_mocked">Using mocked location</string>
     <string name="location_not_mocked">Using actual location</string>
     <string name="pref_tiles_enabled" translatable="false">pref_tiles_enabled</string>
+    <string name="pref_battery_log_enabled" translatable="false">pref_battery_log_enabled</string>
     <string name="pref_tiles_enabled_title">System tiles</string>
+    <string name="pref_tiles_battery_log">Battery Log</string>
     <string name="pref_tiles_enabled_summary">May prevent battery saver apps from completely stopping Trail Sense</string>
+    <string name="pref_tiles_enabled_battery_log_summary">May prevent running battery log every hour in order to accurately show battery drain</string>
```

Comment:
Can you change this to be "Logs battery percentage every hour to estimate time until empty"?